nativelib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -72,6 +75,9 @@ object pthread {
                             stackaddr: Ptr[Byte],
                             stacksize: CSize): CInt = extern
 
+  def pthread_attr_setstackaddr(attr: Ptr[pthread_attr_t],
+                                stackaddr: Ptr[Byte]): CInt = extern
+
   def pthread_attr_setstacksize(attr: Ptr[pthread_attr_t],
                                 stacksize: CSize): CInt = extern
 
```

Comment:
Under X/Open I mostly see `pthread_rwlock_*`, which should be useful for implementing read/write locks, so those are desirable.

Regarding realtime threads—personally, as a possible future user, I would like to see them stay. Since no common Java or Scala concurrency abstractions currently rely on realtime threads, I guess they can be considered low priority. My opinion: keep them if you have them, drop them if they give you trouble.